src/features/collectables/HotspotList.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useCallback, useMemo } from 'react'
+import { times } from 'lodash'
+import { FlatList } from 'react-native-gesture-handler'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import { RefreshControl } from 'react-native'
+import { useNavigation } from '@react-navigation/native'
+import { useTranslation } from 'react-i18next'
+import { useColors } from '../../theme/themeHooks'
+import Box from '../../components/Box'
+import { CollectableSkeleton } from './NftListItem'
+import { Collectable } from '../../types/solana'
+import { CollectableNavigationProp } from './collectablesTypes'
+import HotspotListItem from './HotspotListItem'
+import ButtonPressable from '../../components/ButtonPressable'
+import useHotspots from '../../utils/useHotspots'
+
+const HotspotList = () => {
+  const { bottom } = useSafeAreaInsets()
+  const navigation = useNavigation<CollectableNavigationProp>()
+  const { t } = useTranslation()
+
+  const bottomSpace = useMemo(() => bottom * 2, [bottom])
+  const { primaryText } = useColors()
+  const {
+    hotspots,
+    hotspotsWithMeta,
+    loading: loadingHotspots,
+    refresh,
+  } = useHotspots()
+
+  const flatListItems = useMemo(() => {
+    return hotspotsWithMeta
```

Comment:
should be able to use `hotspotsWithMeta` directly as it's already stable in `useHotspots`

Reply:
Done